Fix progress plugin (#745)
V4Fire/Client: CHANGELOG.md

_Note: Gaps between patch versions are faulty, broken or test releases._

## v3.?.? (20??-??-??)

#### :boom: Breaking Change

> * Removed printLn-progress-view `build/webpack/plugins/progress-plugin`

Reviewer: ```suggestion
* Removed `printLn-progress-view` `build/webpack/plugins/progress-plugin`
```

### :bug: Bug Fix

> * Now webpack progress plugin properly work in webpack watch mode `build/webpack/plugins/progress-plugin`

Reviewer: ```suggestion
* Now `WebpackProgressPlugin` properly work with the Webpack watch mode `build/webpack/plugins/progress-plugin`
```
